<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteCmdDlg</class>
 <widget class="QWidget" name="RemoteCmdDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RemoteCmdDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#frame_5
{
	border-style:solid;
	border-width:0px;
	border-bottom-width:1px;
	border-color:#cccccc;
	background-color:rgb(240,240,240);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_29">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>227</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frameCmd">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame#frameCmd
{
	border-style:solid;
	border-color:#cccccc;
	border-width:1px;
	background-color:white;
}

QFrame
{
	font-family:&quot;Microsoft Yahei&quot;;
	border:none;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_23">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>  同步执行 (直接在网络进程中执行)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_14">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>9999</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_61">
            <item>
             <widget class="QLineEdit" name="edtWinExec">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit
{
	border:1px solid rgb(200,200,200);
	font-family:&quot;Microsoft Yahei&quot;;
	background-color:rgb(240,240,240);
}</string>
              </property>
              <property name="placeholderText">
               <string>输入命令 . . .</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_14">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnWinExec">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	font-family:&quot;Microsoft Yahei&quot;;
	border:1px solid #bbbbbb;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #eeeeee, stop:1 #cccccc);
}

QPushButton:hover
{
	background-color:#d5d5d5;
}

QPushButton:pressed
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #cccccc, stop:1 #eeeeee);
}

QPushButton:disabled
{
	background-color:transparent;
}</string>
              </property>
              <property name="text">
               <string>远程执行</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>1</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame
{
	background-color:#cccccc;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_15">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>9999</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_31">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_8">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_30">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="lbConsoleCaption">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>控制台输入与输出</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbCurrentProcess">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frameExecute">
              <layout class="QHBoxLayout" name="horizontalLayout_66">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="edtConsoleInput">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLineEdit
{
	border:1px solid rgb(200,200,200);
	font-family:&quot;Microsoft Yahei&quot;;
	background-color:rgb(240,240,240);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>输入控制台命令. . . </string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_16">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>10</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btnExecuteConsoleCmd">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>75</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>75</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton
{
	font-family:&quot;Microsoft Yahei&quot;;
	border:1px solid #bbbbbb;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #eeeeee, stop:1 #cccccc);
}

QPushButton:hover
{
	background-color:#d5d5d5;
}

QPushButton:pressed
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #cccccc, stop:1 #eeeeee);
}

QPushButton:disabled
{
	background-color:transparent;
}</string>
                 </property>
                 <property name="text">
                  <string>执行</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_63">
              <item>
               <widget class="QTextEdit" name="edtConsoleOutput">
                <property name="styleSheet">
                 <string notr="true"> QTextEdit#edtConsoleOutput
{
	background-color:black;
	color:white;
	font: 11pt &quot;宋体&quot;;
}
QScrollBar:vertical {
     border-width:0px;
     background;transparent;
     width: 10px;
 }
 QScrollBar::handle:vertical {
     background-color: #555555;
     min-height: 100px;
 }
 QScrollBar::handle:vertical:hover {
     background-color: #666666;
     min-heiht: 100px;
 }
 QScrollBar::add-line:vertical {
	border:none;
	background-color:transparent;
 }
 QScrollBar::sub-line:vertical {
	border:none;
	background-color:transparent;
 }</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="html">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:13pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="acceptRichText">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_62">
              <item>
               <spacer name="horizontalSpacer_17">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btnClearConsoleOutput">
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton
{
	font-family:&quot;Microsoft Yahei&quot;;
	border:1px solid #bbbbbb;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #eeeeee, stop:1 #cccccc);
}

QPushButton:hover
{
	background-color:#d5d5d5;
}

QPushButton:pressed
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #cccccc, stop:1 #eeeeee);
}

QPushButton:disabled
{
	background-color:transparent;
}</string>
                </property>
                <property name="text">
                 <string>清空输出</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_15">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>10</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btnExitProgress">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton
{
	font-family:&quot;Microsoft Yahei&quot;;
	border:1px solid #bbbbbb;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #eeeeee, stop:1 #cccccc);
}

QPushButton:hover
{
	background-color:#d5d5d5;
}

QPushButton:pressed
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #cccccc, stop:1 #eeeeee);
}

QPushButton:disabled
{
	background-color:transparent;
}</string>
                </property>
                <property name="text">
                 <string>强制退出</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_13">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
